speaking-portal-project/src/phonemeFactory/rhubarb.ts

```diff
@@ -42,18 +44,18 @@ export async function rhubarbProcessor(
     const rhubarbProc = spawnSync('./rhubarb', args)
 
     // Return to original directory
+
+    // TODO: Examine chdir error further
```

Comment:
these should never occur correct?

Reply:
The only time I can think of this happening is if the file structure isn't setup properly upon installation or if someone changes the file structure to update Rhubarb (because Rhubarb 2 is currently in development)